src/io.h

```diff
@@ -37,7 +37,7 @@ template <class ThisPtcl> void OutputFileWithTimeInterval(PS::ParticleSystem<Thi
             std::cout << "output " << full_filename << "." << std::endl;
 			std::cout << "//================================" << std::endl;
 		}
-		sysinfo.output_time += end_time / PARAM::NUMBER_OF_SNAPSHOTS;
+        sysinfo.output_time += output_interval;
```

Comment:
While we have not decided on #26 can  you replace these spaces by two tabs? At least keep it consistent where possible.